egs/aishell2/README.md

```diff
@@ -0,0 +1,43 @@
+# AISHELL-2
+
+AISHELL-2 is by far the largest open-source speech corpus available for Mandarin ASR research.
```

Comment:
Please make this a .txt file to match other recipes.
And please don't use the term "open-source" since you are selling the dataset.  Perhaps "free*" would work.
*for research use.
Right now it's not absolutely clear from the sentence that the dataset is only free for non-commercial use... if it is even free for non-commercial use; that isn't absolutely clear to me.